utils/app/const.ts

```diff
@@ -1,5 +1,5 @@
 export const DEFAULT_SYSTEM_PROMPT =
-  "You are ChatGPT, a large language model trained by OpenAI. Follow the user's instructions carefully. Respond using markdown.";
+  process.env.DEFAULT_SYSTEM_PROMPT || "You are ChatGPT, a large language model trained by OpenAI. Follow the user's instructions carefully. Respond using markdown.";
```

Comment:
Add the DEFAULT_SYSTEM_PROMPT in the .env.example file so that it other would know that can add default prompt from environment variable